app/src/main/java/io/gymnasion/cashgroupatms/AtmAdapter.java

```diff
@@ -25,12 +27,13 @@ public AtmViewHolder onCreateViewHolder(ViewGroup parent, int viewType) {
 
     @Override
     public void onBindViewHolder(AtmViewHolder holder, int position) {
-        String name = itemsList.get(position);
+        Atm atm = itemsList.get(position);
```

Comment:
Would be nice to move the binding to a method inside AtmViewHolder, like 

holder.bind(atm);